[IMP] accounting: update the bank accounts documentation (#2417)
odoo/documentation: content/applications/finance/accounting/bank/setup/bank_accounts.rst

allows you to have all your banking data up-to-date and ready for :doc:`reconciliation
<../../bank/reconciliation/use_cases>` with your journal entries.

In Odoo Accounting, each bank account has a dedicated journal set to post all entries in a dedicated
account. Both the journal and the account are automatically created and configured whenever you add
a bank account.

.. note::
   Cash journals and accounts must be configured manually.

Bank journals are displayed by default on the :guilabel:`Accounting Dashboard` in the form of cards
which include action buttons.

.. image:: bank_accounts/card.png
   :align: center
   :alt: Bank journals are displayed on the Accounting Dashboard and contain action buttons

.. _bank_accounts/manage:

Manage your bank and cash accounts
==================================

Connect your bank for automatic synchronization
-----------------------------------------------

To connect your bank account to your database, go to :menuselection:`Accounting --> Configuration
--> Banks: Add a Bank Account`, select your bank in the list, click on :guilabel:`Connect`, and
follow the instructions.

.. seealso::

>    - :doc:`../../bank/feeds/bank_synchronization`

Reviewer: Agreed!

=============
Bank accounts
=============

You can manage as many bank accounts as needed on your database. Configuring them well allows you to
have all your banking data up to date and ready for :ref:`reconciliation <reconciliation/use_cases>`
with your journal entries.

In Odoo Accounting, each bank account is configured to have a dedicated journal set to post all
entries in a dedicated account.

.. note::
   A dedicated journal and a dedicated account are automatically created and configured
   whenever you add a bank account.

The :guilabel:`Bank Journals` are displayed by default on the :guilabel:`Accounting Dashboard` in
the form of convenient cards that and include action buttons.

.. image:: bank_accounts/bank_accounts_card.png
   :align: center

>    :alt: Bank journals are displayed on the Accounting Dashboard and contain actions buttons

Author: here we don't mean parameters, it's really the actions, ie: 1 bill to pay

.. _bank_accounts_add:

Add a bank account
==================

You can either connect your bank account to your Odoo database to have your bank information
synchronized in your database, or you can configure your bank account manually and
:doc:`upload the bank statements manually <../feeds/bank_statements>`.

Bank synchronization
--------------------

To have your bank statements synced automatically, connect your bank account to your database going
to :menuselection:`Accounting --> Configuration --> Add a Bank Account`. Select your bank in the
list and click on :guilabel:`Connect` and follow the instructions.

.. note::
   :doc:`Click here <../../bank/feeds/bank_synchronization>` for more information on bank
   synchronization.

Manual configuration
--------------------

If your bank institution can’t be synchronized automatically, or if you prefer not to sync it with

> your database, you may also configure your bank account manually.

Author: I updated as per most of your comments

If your bank institution can’t be synchronized automatically, or if you prefer not to sync it with
your database, you may also configure your bank account manually.

To add a new bank account manually, go to :menuselection:`Accounting --> Configuration`, click on
*Add a Bank Account*, then on *Create it*, and fill out the form.
One bank journal is already created by default in your Odoo Accounting and is available from
:menuselection:`Configuration --> journals`. Open it to edit the different information related to
your bank account.

- **Name**: the bank account's name, as displayed on Odoo.
- **Account Number**: your bank account number (IBAN in Europe).
- **Bank**: click on *Create and Edit* to configure the bank's details. Add the bank institution's
  name and its Identifier Code (BIC or SWIFT).
- **Code**: this code is your Journal's *Short Code*, as displayed on Odoo. By default, Odoo creates
  a new Journal with this Short Code.
- **Journal**: This field is displayed if you have an existing Bank Journal that is not linked yet
  to a bank account. If so, then select the *Journal* you want to use to record the financial

>   transactions linked to this bank account or create a new one by clicking on *Create and Edit*.

Author: Not sure it is really needed to describe those fields, and a short description of the fields is available when moving the computer mouse over it.

> If you need to **edit your bank account details**, click on the :guilabel:`Account number`. A new

Author: no you really have to click on the bank account 
![image](https://user-images.githubusercontent.com/85342080/177983887-bdb2b995-a4b2-473d-b31c-d617ed03891c.png)
![image](https://user-images.githubusercontent.com/85342080/177983948-151f1c12-43e0-4522-96e6-e2a70b65d80c.png)

> The :guilabel:`Bank Feeds` can be selected from the bank journal. 3 options are possibles:

Reviewer 2: ```suggestion
:guilabel:`Bank Feeds` can be selected from the bank journal. Three options are available:
```
Numbers from zero to nine are usually spelled out. Always spell out numbers when they start a sentence.

> - :guilabel:`Undefined yet`, should be selected if you don’t know yet if you will sync your bank
>   with Odoo or not.
> - :guilabel:`Import (CAMT,CODA, CSV, OFX, QIF)`, should be selected if you want to import your bank
>   statement via a different format.
> - :guilabel:`Automated Bank Synchronization`

Reviewer 2: ```suggestion
- :guilabel:`Undefined yet`, which should be selected when you don’t know yet if you will sync your bank account
  with your database or not.
- :guilabel:`Import (CAMT, CODA, CSV, OFX, QIF)`, which should be used if you want to import your bank
  statements using a different format.
- :guilabel:`Automated Bank Synchronization`
```
It feels weird that you don't explain the last point, even if obvious.

=============
Bank accounts
=============

You can manage as many bank accounts as needed on your database. Configuring them well allows you to
have all your banking data up-to-date and ready for :doc:`reconciliation
<../../bank/reconciliation/use_cases>` with your journal entries.

In Odoo Accounting, each bank account has a dedicated journal set to post all entries in a dedicated
account. Both the journal and the account are automatically created and configured whenever you add
a bank account.

> :guilabel:`Bank journals` are displayed by default on the :guilabel:`Accounting Dashboard` in

Reviewer 2: ```suggestion
:guilabel:`Bank journals` are displayed by default on the :guilabel:`Accounting Dashboard` in
```
Do `Bank journals` appear on the UI? (just asking, if not, no guilabel ;))

the form of cards which include action buttons.

.. image:: bank_accounts/bank_accounts_card.png
   :align: center
   :alt: Bank journals are displayed on the Accounting Dashboard and contain action buttons

.. _bank_accounts_add:

Add a bank account
==================

You have two options to add a bank account:

- Connect it to your database to synchronize your bank statements automatically.
- Create it and :doc:`upload your bank statements manually <../feeds/bank_statements>`.

Connect your bank for automatic synchronization
-----------------------------------------------

To connect your bank account to your database, go to :menuselection:`Accounting --> Configuration
--> Banks: Add a Bank Account`. Select your bank in the list, click on :guilabel:`Connect`, and
follow the instructions.

.. seealso::

>    :doc:`../../bank/feeds/bank_synchronization`.

Reviewer 2: ```suggestion
   :doc:`../../bank/feeds/bank_synchronization`
```
Not a sentence, full stop isn't needed

Create a bank account
---------------------

If your banking institution is not available in Odoo, or if you don't want to connect your bank
account to your database, you can configure your bank account manually.

> A default bank journal is available and can be used to configure your bank account, by going to

Reviewer 2: ```suggestion
A default bank journal is available and can be used to configure your bank account by going to
```
No comma

.. image:: bank_accounts/bank_account_manual_config.png
   :align: center
   :alt: Manually configure your bank information

To manually add another bank account, go to :menuselection:`Accounting --> Configuration --> Banks:
Add a Bank Account`, click on :guilabel:`Create it` (at the bottom right), and fill out the form.

.. note::
   Odoo automatically detects the bank account type (e.g., IBAN) and enables some features
   accordingly.

.. _bank_accounts_configuration:

Edit an existing bank account
=============================

To edit an existing bank account, go to :menuselection:`Accounting --> Configuration --> Accounting:
Journals` and select the bank journal you want to modify.

If you need to **edit your bank account details**, click on your :guilabel:`Account Number`. On the
new page, click on your :guilabel:`Bank` and update your bank information accordingly. These details
are used when registering payments.

.. image:: bank_accounts/edit_your_bank_account.png
   :align: center
   :alt: Edit your bank information

:guilabel:`Bank Feeds` can be selected from the bank journal to define how your bank statements are
registered. Three options are available:

> - :guilabel:`Undefined yet`, which should be selected when you don’t know yet if you will synchronize your

Reviewer 2: Line break after 100 characters
